src/ui/public/agg_types/metrics/std_deviation.js

```diff
@@ -11,19 +11,18 @@ export default function AggTypeMetricStandardDeviationProvider(Private) {
       return details.valProp;
     },
     makeLabel: function () {
-      let details = this.keyedDetails(this.params.customLabel)[this.key];
-      return details.title + ' of ' + this.fieldDisplayName();
+      return this.keyedDetails(this.params.customLabel, this.fieldDisplayName())[this.key].title;
```

Comment:
Yeah, I thought about breaking it up too, for better readability. Thanks for nudging me over the fence :)

I'll use lodash's get function but could you educate me as to why that's preferable?